### ISSUE TITLE
D2BS is not responding... starting profile

### ISSUE THREAD

**Issue by Klainx** [kolton/d2bot-with-kolbot#874]
Hello guys,

After so many try and research i'm still in the blue...
I've created a folder with fresh kolton bot updated svn https://github.com/kolton/d2bot-with-kolbot/trunk At revision: 18152 right now.
I even made a new fresh install of D2 with it updated before getting started.
Put every things needed to be in administrator mode + compatibility mode
Created my profile in D2bot with everything done correctly.
I also have the microsoft ret package 2008, 2010, 2012, 2013, 2015, 2017 and Framework.net 4.7.2 already installed.

But no matter what when i start the bot profile the game start wait in few seconds and then crash to finally restart and looping like that till i stop it.

The console only show me this message

D2BS is not responding... starting profile

Im actually Running on Windows 10 Pro x64

Thank you in advance for any assistance !

**Comment by 5noop** [comment]
Please post d2bs.log on pastebin.com, or at least the end of it.

**Comment by Klainx** [comment]
https://pastebin.com/2Kysu9pr

**Comment by 5noop** [comment]
Ok then increasing the response time will probably not change anything.

What you could try, if you haven't done it yet:
- reinstall the dependencies
- use the limedrop-utf8 branch instead of trunk

**Comment by Klainx** [comment]
Yeah i've already done the uninstall/reinstall part and response tim, but nothing.

And yeah i've got both Kolton and Limedrop, Limedrop work well. just that i seem to need to create a game every time to have him doing a run.
